### ISSUE TITLE
Config/secret list sort

### ISSUE THREAD

**Issue by mrspartak** [swarmpit/swarmpit#472]
This is just a suggestion to sort by name all configs and secrets
![image](https://user-images.githubusercontent.com/993910/75151360-91c9b080-5717-11ea-86ae-44b743fe8957.png)

**Comment by nohaapav** [comment]
btw you can get to wanted resource by typing

**Comment by mrspartak** [comment]
Yes, that is what I do for now. I think this feature will solve it partly too
https://github.com/swarmpit/swarmpit/issues/255
